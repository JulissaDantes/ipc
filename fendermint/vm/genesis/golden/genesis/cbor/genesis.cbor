a86a636861696e5f6e616d65662053487f357c6974696d657374616d701b9201bf0d8f24ea556f6e6574776f726b5f76657273696f6e1568626173655f66656551001283038f8c95fdc6e879981a74f910a96b706f7765725f7363616c65206a76616c696461746f727381a26a7075626c69635f6b65795841049d5903e6c0c62605882883f2c36587799e78c062e30b600d36dd5a1cab4a6e2b502717fae26080973cd3652001f8d3443566d88ec613bf0868b2866fb017ebcd65706f776572a26a636f6c6c61746572616c5100edb0eb8a097f2754f9efbc3d1e26fc366f6665646572617465645f706f7765724a000108b257ea7b795f0c686163636f756e7473807365616d5f7065726d697373696f6e5f6d6f6465a1646d6f64656c756e72657374726963746564