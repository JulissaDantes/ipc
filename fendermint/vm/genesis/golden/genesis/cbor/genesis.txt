Genesis { chain_name: " SH\u{7f}5|", timestamp: Timestamp(10520900269471230549), network_version: NetworkVersion(21), base_fee: TokenAmount(24606367032961828376.455539334323966121), power_scale: -1, validators: [Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [55209515, 56002346, 33608557, 25922605, 27162816, 14246366, 42483500, 9967136, 65454278, 2577984], magnitude: 1, normalized: true }, y: Field { n: [1567693, 10591212, 66094731, 37427278, 3499736, 8271057, 20337152, 33709299, 66773600, 1313221], magnitude: 1, normalized: true }, infinity: false })), power: GenesisPower { collateral: TokenAmount(315945656550809561059.164174190540946486), federated_power: TokenAmount(19.073404036472987404) } }], accounts: [], eam_permission_mode: Unrestricted, ipc: None }